operator/v1/types_klusterlet.go

```diff
@@ -86,6 +86,22 @@ type KlusterletSpec struct {
 	// registration-agent and work-agent will use it to communicate with hub api server.
 	// +optional
 	HubApiServerHostAlias *HubApiServerHostAlias `json:"hubApiServerHostAlias,omitempty"`
+
+	// ResourceType specify QoS classes of klusterlet deployment
+	// +optional
+	ResourceType ResourceType `json:"resourceType,omitempty"`
```

Comment:
Not sure ResourceType is a good name. How about `ResourceRequest` or `ResourceRequirement`?

Reply:
ok. let's use ResourceRequirement